mongo/datadog_checks/mongo/config.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, instance, log):
         else:
             self.server = None
             self.hosts = instance.get('hosts', [])
+            if type(self.hosts) == str:
```

Comment:
This class handles the integration config. We started adopting this model before the config specs existed but also, unlike the config models, this is PY2 compatible so it is actually what it is used in code.